tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -755,6 +776,33 @@ impl<F, T> Format<F, T> {
         }
     }
 
+    /// Sets whether or not the binary name of the process is displayed when
+    /// formatting events. If binary_name is None, argv\[0\] will be used,
+    /// otherwise the spcified string will be used.
+    ///
+    /// [argv\[0\]]: std::env::args
+    pub fn with_binary_name(self,
+                            display_binary_name: bool,
+                            binary_name: Option<&str>,
+    ) -> Format<F, T> {
+        Format {
+            display_binary_name,
+            binary_name: binary_name.map(str::to_string),
```

Comment:
This here could then be `.map(Into::into)`.

Reply:
Done.